.travis.yml

```diff
@@ -17,10 +17,10 @@ notifications:
 git:
   depth: 99999999
 
-## Debug documentation in travis
 env:
   global:
-    - DOCUMENTER_DEBUG=true
+    - DOCUMENTER_DEBUG=true ## Debug documentation in travis
+    - JULIA_NUM_THREADS=2 ## run Julia on two threads
```

Comment:
Do you know if there is a *programmatic* way of changing the number of threads? I am thinking in a way to avoid setting an ENV variable...

Reply:
I'm not aware of a programmatic way to set it programatically... Currently Julia reads the number of threads from the env variable `JULIA_NUM_THREADS`, which must be set *before* Julia starts. It can be set from within Julia, but then the Julia session should be restarted so that the changes take effect.

Reply:
Thanks for the clarification!